src/SprykerEco/Yves/Payone/PayoneConfig.php

```diff
@@ -13,6 +13,18 @@
 
 class PayoneConfig extends AbstractBundleConfig
 {
+    protected const PAYONE_BANCONTACT_AVAILABLE_COUNTRIES = [
+        'BE' => 'Belgium',
+    ];
+
+    /**
+     * @return array
```

Comment:
`* @return string[]`

I would also describe its structure, but that's optional:
```
* @return string[] Keys are country codes
```